projects/aca-content/src/lib/ui/variables/variables.scss

```diff
@@ -35,6 +35,10 @@ $action-button-text-color: rgba(33, 35, 40, 0.7);
 $page-layout-header-background-color: #fff;
 $search-chip-icon-color: #757575;
 $disabled-chip-background-color: #f5f5f5;
+$adf-metadata-property-panel-border-color: rgba(0, 0, 0, 0.12);
```

Comment:
this is ACA, why the prefix is "adf-" ? are you rewriting the ADF variables?

Reply:
Updated the variable names.